packages/server-wallet/src/wallet/chain-listener.ts

```diff
@@ -0,0 +1,47 @@
+import {makeDestination} from '@statechannels/wallet-core';
+
+import {ChainEventSubscriberInterface} from '../chain-service';
+import {Engine} from '../engine';
+import {Store} from '../engine/store';
+import {MessageServiceInterface} from '../message-service/types';
+import {getMessages} from '../message-service/utils';
+import {Uint256} from '../type-aliases';
+
+export function createChainListener(
+  engine: Engine,
+  store: Store,
```

Comment:
Is this parameter used?

Reply:
Nope, since we're pulling off the store from the `engine.